docs/manage/manage-subscription/fedramp-capabilities.md

```diff
@@ -0,0 +1,108 @@
+---
+id: fedramp-capabilities
+title: Sumo Logic FedRAMP Solution Capabilities
+sidebar_label: FedRAMP Capabilities
+description: Learn the capabilities included with Sumo Logic’s FedRAMP solution.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Sumo Logic is FedRAMP® Moderate Authorized, providing the observability and security that federal agencies and commercial entities require to ensure their applications are reliable and secure. For more information, see the [FedRAMP Authorization](https://www.sumologic.com/fedramp/) page on our Sumo Logic site.
+
+For information about the FedRAMP certification of Sumo Logic’s solution, see [Sumo Logic in the FedRAMP Marketplace](https://marketplace.fedramp.gov/products/FR1918740338) (package ID FR1918740338). 
+
+The following table shows the capabilities included with Sumo Logic’s FedRAMP solution compared to Sumo Logic's commercial product.
```

Comment:
_Sumo Logic's commercial product_ <-- is this referring to the Sumo Logic Logic platform in general? It looks like [here](https://www.sumologic.com/fedramp/) it's referred to as _Continuous Intelligence Platform™_.

Reply:
You're right, @kimsauce. "Commercial" is not a good term. I carried that over from the source document. So I changed it to "standard". And "CIP" won't work, because it's actually CIP that is FedRAMP certified.